specification/http/src/main/scripts/org/kaazing/specification/http/rfc7232/headers/get.request.with.if.modified.since.header/request.rpt

```diff
@@ -0,0 +1,24 @@
+connect http://localhost:8000/resource
+connected
+
+write method "GET"
+write version "HTTP/1.1"
+write header content-length
+write flush
+
+read status "200" "OK"
+read header "ETag" "abc123"
+read header "Last-Modified" /.+/
```

Comment:
we should capture the ETag header value as a regular expression and use that captured value in later request for same resource to check if content has been modified or not.